src/app/app.component.ts

```diff
@@ -1,10 +1,28 @@
 import { Component } from '@angular/core';
+import { Router, RouterModule } from '@angular/router';
+import { SharedDataService } from './services/shared.data.service'
 
 @Component({
     selector: 'app-root',
     templateUrl: './app.component.html',
     styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
+    private static _localStorageKey = 'pcprepkitUser';
     title = 'PC Prep Kit';
+    public position = 'col-md-10 col-md-offset-2';
+    public loggedIn = false;
+    
+    constructor(private _router: Router, private _sharedData: SharedDataService) {
+        this._router.events.pairwise().subscribe((e) => {
+            this.loggedIn = (e && localStorage.getItem(AppComponent._localStorageKey)) ? true : false;
```

Comment:
There was a typescript issue with this as it returns the type of the second operand  (https://stackoverflow.com/questions/12693787/why-does-the-operator-produce-the-type-of-the-second-operand) and in this case localStorage.getItem(AppComponent._localStorageKey) is a string so since this.loggedIn is implicitly defined as a boolean, it throws an error. I seem to have missed it while testing.